### DISCUSSION TITLE
android4.4.2

### DISCUSSION THREAD

**Discussion by wentywenty** [hushenghao/AndroidEasterEggs#31] [Q&A]
好像没有这个

**Comment by wentywenty** [comment]
![IMG_20230805_140738.jpg](https://github.com/hushenghao/AndroidEasterEggs/assets/63338034/9c293178-10be-4125-922d-54b2af9b3bc4)

[]()![IMG_20230805_140731.jpg](https://github.com/hushenghao/AndroidEasterEggs/assets/63338034/19e780a2-d4ad-4dc3-b78d-cc47b1a60b23)

![IMG_20230805_140723.jpg](https://github.com/hushenghao/AndroidEasterEggs/assets/63338034/b58e3c7d-bb54-4dea-aa0e-4544f9fecc36)

**Comment by hushenghao** [comment]
并没有什么不同，只是小米将build Id给改成了H开头了。
android 4.4彩蛋默认就是K，代表KitKat，和小版本没关系

**Reply by wentywenty** [reply]
thankyou
